README.md

```diff
@@ -98,6 +108,16 @@ Create a new lion instance with the name `simba`
 
 ```ruby
 # code here
+  class Animal
+    attr_accessor :name
+    def intialize(name,greet)
+      @name = name
+      def greet
+        puts @name
+      end
```

Comment:
`greet` should not be defined inside of `initialize`. It should be its own separate method inside of the class.